### DISCUSSION TITLE
中國的社會信用制度

### DISCUSSION THREAD

**Discussion by 109302013evantsao** [1101-GS4719-C/Python-Community-GS4719-C#238] [General]
中國的社會信用制度是建立在大數據和演算法的基礎上實施的，透過監視器去對每一個人做評分
每個人都有一個數字，會依照你做好事或做壞事而增減，分數高的人會受到政府表揚
，分數低的人會被限制自由、遭人嫌棄、得到不平等待遇等等，
我認為這個制度有好又壞，我不認為一個人能夠以一個分數去做評量，更不用說這個評分標準是由
國家政府而規畫出來的，毫無通融可能性，也不妨有些有心人士假裝做好事，
我覺得這個系統比較適合使用在自動搜尋犯罪人士(已經有案底，逃犯)
不應該使用這個系統去壓迫人民，使人民時時擔心自己有沒有做錯事情
[相關新聞](https://www.gvm.com.tw/article/80930)

**Comment by HSU-YU-JEN** [comment]
完全同意，感覺演算法這個技術讓集權政府更容易控制與限制人民

**Comment by xCk27x** [comment]
我覺得此制度完全不合理，社會信用制度直接把  **更生人** 這個本就存在的社會問題加廣到了全民身上。刑法已經明確規定了犯法將會得到的處罰，結果在得到應有的處罰後還要繼續被社會歧視，這跟引進150年前的黑奴制度有甚麼差別?

**Comment by 107602505** [comment]
記得有一部影集叫黑鏡就是跟這個有關的，有興趣的可以去看看~
https://www.hk01.com/%E5%8D%B3%E6%99%82%E5%A8%9B%E6%A8%82/352157/%E9%BB%91%E9%8F%A1-%E7%A5%9E%E9%A0%90%E8%A8%80-%E7%A4%BE%E6%9C%83%E4%BF%A1%E7%94%A8%E9%AB%94%E7%B3%BB-%E6%B8%AF%E4%BD%9C%E5%AE%B6%E6%89%B9-%E6%94%BF%E5%BA%9C%E7%B6%AD%E7%A9%A9%E7%9A%84%E6%9C%80%E9%AB%98%E7%9B%AE%E6%A8%99
這篇報導有提到「社會信用系統」是讓人民去討好政府訂的道德標準，當人們一生追求更高分數時，他自然不會追求其他人活著的目的，這就是極權政府維穩的最高目標，此制度無疑是集權政府管制人類的方法，雖然人民為了分數不會做出脫序的行為但我覺得人性也會漸漸消失。

**Comment by MAKERRRofKEVIN** [comment]
個人淺見認為社會信用制度的最大問題在於如何判定善惡，人工審理的好處在於法官的自由心證能夠從人的角度出發作出更人性的判決，機器儘管可以達到公平，卻未必能達到人性

**Comment by wax75397** [comment]
不論如何，中國獲得監視每個國民的能力是非常的可怕的。甚麼隱私都沒有，每天都要擔驚受怕的生活，深怕做出一些事而「被消失」。

**Comment by Isongzhe** [comment]
我想到之前美國脫口秀調侃社會信用制度，你是台灣人 直接0分

**Reply by ChinChun0220** [reply]
對這個影片有印象，是有個人突然出現說自己是台灣人就變成0分XD

**Comment by mesakiyuta** [comment]
生活在處處都被監視、被計算點數的社會裡，感覺心理壓力好大，必須每一刻都要注意自己的行為，最可怕的是這社會信用評分會嚴重影響到未來，想想就覺得很不舒服。

**Comment by YuHengChen54** [comment]
連只是去買個東西走在路上都要被監視，真的很變態

**Comment by www0000596** [comment]
但感覺中國政府如果真的要實行，大部分人可能也會選擇接受？（避免對抗政府的權威？）
